roles/reproducer/tasks/reuse_main.yaml

```diff
@@ -0,0 +1,274 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Load CI job environment
+  tags:
+    - bootstrap_layout
+  when:
+    - cifmw_job_uri is defined
+  ansible.builtin.include_tasks:
+    file: ci_data.yml
+    apply:
+      tags:
+        - bootstrap_layout
+
+- name: Discover and expose CI Framework path on remote node
```

Comment:
This is the same code, which you already provided in the `configure_cleanup.yaml`. Could it be unified?

Reply:
As those are separate task files that might not always be called together I would like to keep them independent of each other to avoid issues in running only one of them